.travis.yml

```diff
@@ -1,22 +1,20 @@
-language: rust
-rust:
-  - stable
-  - nightly
-
+language: node_js
+node_js:
+  - 12.0
 env:
-  - NODE_ENV=ci
-  - NODE_ENV=ci-staging
-cache: yarn
-
+  - NODE_ENV=ci YARN_GPG=no
+  - NODE_ENV=ci-staging YARN_GPG=no
+os:
+  - linux
+  - osx
+  - windows
+git:
+  autocrlf: false
 before_install:
+  - curl https://sh.rustup.rs -sSf -m 60 | sh -s -- -y -v --default-toolchain stable
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then HOME="/c/Users/travis"; fi
+  - export PATH="$HOME/.cargo/bin:$PATH"
   - rustup target add wasm32-unknown-unknown
-  - nvm install 12
-  - nvm use 12
-  - npm install -g yarn
-
-install: 
-  - yarn install
-
 script:
-  - yarn test
-  - yarn lint
+  - npm run test
```

Comment:
why use npm? Is it faster on CI?

Reply:
there was a big issue with Yarn when [running tests on Windows](https://travis-ci.community/t/build-timeout-but-actually-succeeded/2763), so part of this was to attempt to avoid it. Then I looked to see if there would be issues with caching as we've noticed `yarn` seems to work well during workshops. So [I looked into it](https://docs.travis-ci.com/user/caching/#npm-cache) and saw that it's cached by default on Travis.
At this point it seems like we could go either way. Do you want me to change it back, @vgrichina ?

Reply:
On this subject, this is a good time to mention that this is why we have `YARN_GPG=no` in the Travis config. I tried every which way and indeed this is an issue with Windows and Travis, as I found in a few places online. Wanted to give you the heads up on that, as it's a flag that'll be seen in some examples, too.